challenges/qr_code_component/index.html

```diff
@@ -54,7 +71,7 @@
       <div class="mt-3">
         <a
           class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA/frontend_mentor_challenges/tree/qr_code_component/challenges/qr_code_component"
+          href="https://github.com/frontend_mentor_challenges/tree/qr_code_component/challenges/qr_code_component'"
```

Comment:
You need to put your username, so visitors can access your code for the challenge.